### PR TITLE
Add support for templating in init-container

### LINKED ISSUE #141: Add yaml support to init container
Generating yaml configurations is becoming popular, in my case I found the need for it because that is the format [rtmbot](https://github.com/slackapi/python-rtmbot/blob/master/docs/example-config/rtmbot.conf) is expecting.

Adding support for this in Kamus can ease onboarding for many services, so I'll open a PR for it. 
WDYT?

### PR DESCRIPTION
**Pull request by omerlh**
close #141 #202

### CODE REVIEW

File: init-container/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+kubectl get secret $(kubectl get sa default -o jsonpath='{.secrets[0].name}') -o jsonpath='{.data.token}' | base64 -D > token.txt
+```
+4. Set the following env var for the init container to work correctly:
+```
+export set TOKEN_FILE_PATH=$(pwd)/token.txt
+export set KAMUS_URL=http://localhost:5000
+```
+5. Now you can run the init container:
+```
+node index.js -e encrypted -d decrypted -n output.json
```
Comment (shaikatz) [inline init-container/CONTRIBUTING.md]: Explain what is `encrypted` ? is a folder? is it a file?

File: init-container/README.md [hunk 2]
```diff
@@ -28,4 +28,36 @@ The init container accept the following environmenmt variables:
...
+This will result in the following file created by the init container:
+```
+<decrypted value>
+hello
+```
+
+Look on EJS docummentation for more details, or on one of the existing [templates](https://github.com/Soluto/kamus/tree/master/init-container/templates) for ideas on how you can use it. The template input is:
+```
+{
+  "secrets": [] //array of the decretyped items, key value pairs.
```
Comment (shaikatz) [inline init-container/README.md]: Unclear what is `the template input is` ?

File: init-container/CONTRIBUTING.md (hunk 1 above)
Comment (omerlh) [reply init-container/CONTRIBUTING.md]: it's a folder - I put it there with some ready-made encrypted content that is ready for debugging

Comment (shaikatz) [reply init-container/CONTRIBUTING.md]: I mean, it needs to be more clear in the MD.

**Pull request merged**